phlib/include/phnativeinl.h

```diff
@@ -724,7 +724,8 @@ FORCEINLINE
 NTSTATUS
 PhGetProcessIsXFGuardEnabled(
     _In_ HANDLE ProcessHandle,
-    _Out_ PBOOLEAN IsXFGuardEnabled
+    _Out_ PBOOLEAN IsXFGuardEnabled,
+    _Out_opt_  PBOOLEAN IsXFGuardAuditEnabled
 )
```

Comment:
) doesn't align with _Out_opt_

Reply:
Oh..at first I wanted to make the last parameter Optional but then I changed my mind, SAL is bad in this case...I'll fix it, don't merge yet please

Reply:
> don't merge yet please

Is is ok to marge?

Reply:
Yes, thanks.